test/test_AutoindexHandler.cpp

```diff
@@ -0,0 +1,38 @@
+#include "gmock/gmock.h"
```

Comment:
Should be deleted. <gmock/gmock.h> is already included

Reply:
ture, removed